react/javascript/demo-external-api/manifest.lkml

```diff
@@ -0,0 +1,25 @@
+project_name: "demo-external-api"
+
+application: demo-external-api {
+  label: "demo-external-api"
+  # use url for development
+  url: "http://localhost:8080/bundle.js"
+  # use file for production deployment
+  # file: "bundle.js
+  entitlements: {
+    local_storage: no
+    navigation: no
+    new_window: no
+    new_window_external_urls: []
+    use_form_submit: yes
+    use_embeds: no
+    use_downloads: no
+    use_iframes: no
+    use_clipboard: yes
```

Comment:
Extra entitlement? nbd just got me wondering "where in this is the clipboard leveraged?"

Reply:
Also, why bother leaving in all of the `no` entitlements? Is there a difference between an omission and an explicit no?

Reply:
Fixed. I like to show all available entitlements so that they can see what is available.